site/src/api/queries/experiments.ts

```diff
@@ -1,15 +1,13 @@
-import type { UseQueryOptions } from "react-query";
 import * as API from "api/api";
 import type { Experiments } from "api/typesGenerated";
-import { getMetadataAsJSON } from "utils/metadata";
+import type { MetadataState } from "hooks/useEmbeddedMetadata";
 import { cachedQuery } from "./util";
 
-const initialExperimentsData = getMetadataAsJSON<Experiments>("experiments");
 const experimentsKey = ["experiments"] as const;
 
-export const experiments = (): UseQueryOptions<Experiments> => {
+export const experiments = (metadata: MetadataState<Experiments>) => {
```

Comment:
I do want to go back and add explicit return types, but I was on a tight deadline, and didn't have time.

These functions are still type-safe and will let you know if you wire things wrong. They're just not "add explicit type parameter"-safe, because of TypeScript generic shenanigans